src/lock_file/resolve/resolver_provider.rs

```diff
@@ -158,7 +159,15 @@ impl<'a, Context: BuildContext> ResolverProvider for CondaResolverProvider<'a, C
             .right_future()
     }
 
-    fn with_reporter(self, reporter: impl uv_distribution::Reporter + 'static) -> Self {
+    // TODO: Actually implement this....
```

Comment:
@tdejager, Do we need to do anything with the conda installed packages?

Reply:
Because the metadata should be readable by uv this should work out I think.